docs/topics/auth/customizing.txt

```diff
@@ -299,6 +299,35 @@ example, the following checks if a user may view tasks::
 
     user.has_perm('app.view_task')
 
+Custom permissions may contain the ``%(class)s`` and ``%(verbose_name)s``
+string. Child models will then inherit the specific permission::
```

Comment:
I'm not sure how to word this, technically the interpolation should always be usuable, however you can only inherit the `Meta` class from abstract models, and it doesn't make much sense to use the interpolations form on non-abstract models.